apps/ui/src/components/FormAuctionBid.vue

```diff
@@ -17,6 +17,7 @@ import { getValidator } from '@/helpers/validation';
 
 const DEFAULT_PRICE_PREMIUM = 1.001; // 0.1% above clearing price
 const PRICE_DECIMALS = 4;
+const MAX_PRICE_DECIMALS = 6;
```

Comment:
This value is hardcoded to `6` on the gnosis auction app, not sure why it's not dynamic, and following the auction bidding token decimal